src/PrestaShopBundle/Resources/views/Admin/Product/CatalogPage/Lists/list.html.twig

```diff
@@ -23,117 +23,118 @@
  * International Registered Trademark & Property of PrestaShop SA
  *#}
 <tbody
-  {% if activate_drag_and_drop and has_category_filter %}class="sortable"{% endif %}
-  last_sql="{{ last_sql_query|escape('html_attr') }}"
+    {% if activate_drag_and_drop and has_category_filter %}class="sortable"{% endif %}
+    last_sql="{{ last_sql_query|escape('html_attr') }}"
```

Comment:
Maybe should be a `data-` attribute?

Reply:
nope! we dont change anything here: not the scope of this contribution. 

HALTE AU GRABUGE !